jest.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  projects: ['<rootDir>'],
```

Comment:
for reference: [`projects` configuration docs](https://jestjs.io/docs/en/configuration#projects-arraystring--projectconfig)

The setting doesn't seem to make any difference, and is unrelated to the other changes in this PR, could you remove it? [We can discuss it](https://github.com/pikapkg/snowpack/discussions/) separately if you think it's worth adding it?

Reply:
Removed, forgot to revert it back, it was added when I was trying something else.